packages/fxa-shared/test/lib/logger.ts

```diff
@@ -0,0 +1,116 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { assert } from 'chai';
+import Logger from '../../lib/logger';
+import sinon = require('sinon');
+
+describe('lib/logger', () => {
+  let logger: Logger;
+
+  describe('constructor', () => {
+    it('should create logger', () => {
+      sinon.spy(window.console, 'info');
+
+      const information = 'Hello Firefox!';
+
+      logger = new Logger();
+      logger.info(information);
+
+      assert.isTrue(
+        (window.console.info as sinon.SinonSpy).calledOnceWith(information)
+      );
+      (window.console.info as sinon.SinonSpy).restore();
```

Comment:
My preference's been assigning the spy into a const and then I can restore on it without type assertions.